judges/github-events/github-events.rb

```diff
@@ -43,6 +43,10 @@ def self.skip_event(json)
     raise Factbase::Rollback
   end
 
+  def self.reviewer?(json)
```

Comment:
@Suban05 it's not a good idea in general to make methods with one line just to make code more readable. Instead, you can use a boolean variable.